docs/superpowers/plans/2026-08-15-legal-source-and-pages.md

```
@@ -0,0 +1,672 @@
+corepack pnpm --filter @markiro/legal-documents test
+```
+
+Expected: package is selected and the test import fails because the registry
+source does not exist.
+
+- [ ] **Step 3: Add the package and minimal validator**
+
+Use pure deterministic validation. Parse revisions with
+`/^(\d{4})\.(0[1-9]|1[0-2])\.(\d{2})$/`, validate dates by round-tripping
+`new Date(`${value}T00:00:00.000Z`)`, reject unexpected external routes, and
+build keys with `${code}/${revision}`. Do not use broad casts or `any`.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)